src/doc-utils.js

```diff
@@ -126,11 +126,19 @@ function propagateBreaks(doc) {
   );
 }
 
+function getNodeSource (node, text) {
+  return text
```

Comment:
In utils, there's `locStart` and `locEnd` you can use to get the proper position.